src/RageUtil.cpp

```diff
@@ -124,6 +124,16 @@ float fmodfp(float x, float y)
 
 int power_of_two( int input )
 {
+	if (input == 0)
+	{
+		return 0;
+		LOG->Trace("power_of_two: input is %d, returning 0", input);
+	}
+	if (input < 0)
+	{
+		LOG->Trace("power_of_two: input is %d, returning -1", input);
+		return -1;
+	}

```

Comment:
This is straight up wrong use and can be changed to an assert to quickly catch that case if it ever, tough very unlikely, happens